uint32 msg_cmd
uint64 msg_Id
uint32 msg_subId
uint32 msg_pubId
uint64 time
string data